test/chplcheck/SKIPIF

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+source $CHPL_HOME/util/config/run-in-venv-common.bash
+
+python3 -c "import chapel" 2> /dev/null
+if [ $? -eq 0 ] ; then
+echo "False"
+else
+echo "True"
+fi
+
```

Comment:
```suggestion
if python3 -c "import chapel" 2> /dev/null; then
    echo "False"
else
    echo "True"
fi

```

Definitely indent the stuff inside the `if`, and consider skipping `$?` in favor of just putting the command into the if.

Reply:
for sure.